src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -128,6 +129,34 @@ public boolean link() throws IOException, InterruptedException {
             return false;
         }
 
+        // Prepare files
+        String sourceOS = projectConfiguration.getTargetTriplet().getOs();
+        Path rootPath = paths.getSourcePath().resolve(sourceOS);
```

Comment:
What does this directory resolve to, relative to the root directory?

Reply:
that is `/path/to/src/android` in this case

Reply:
OK, sounds a bit strange to use `sourceOS` as variable name. `targetOS` sounds more clear?

Reply:
`source` comes from `src`, and `OS` is for the targetOS, like `android` or `ios`(We use the same in `utils/ios/InfoPlist`).

Reply:
Yes, but the thing is, `sourceOS` is a variable that holds the name of the target OS and has not yet anything to do with `src/...`. It's only used for determining the `rootPath`, which should actually be `path_to_target_OS_sources`:

    String targetOs = projectConfiguration.getTargetTriplet().getOs();
    Path targetSourcePath = paths.getSourcePath().resolve(targetOs);

Reply:
Ok, changed it.